Loop/View Models/SimpleBolusViewModel.swift

```diff
@@ -0,0 +1,332 @@
+//
+//  SimpleBolusViewModel.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 9/29/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import LoopKit
+import LoopKitUI
+import os.log
+import SwiftUI
+import LoopCore
+import Intents
+import LocalAuthentication
+
+protocol SimpleBolusViewModelDelegate: class {
+    
+    ///
```

Comment:
[nit] Why these empty `///`?

Reply:
Hmm, I think I got them from BolusEntryViewModelDelegate. I can clean them up in both places.